src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts

```diff
@@ -451,8 +451,10 @@ export class ParameterHintsWidget implements IContentWidget, IDisposable {
 
 	next(): boolean {
 		const length = this.hints.signatures.length;
+		const last = (this.currentSignature % length) === (length - 1);
 
-		if (length < 2) {
+		// If there is only one signature, or we're on last signature of list
+		if (length < 2 || last) {
```

Comment:
You might as well update the lower bits in these functions because now you won't ever reach them with the spill over/under.

Reply:
Good point! Will update.

Reply:
@jrieken okay - how's that?